src/deploy.py

```diff
@@ -24,13 +27,19 @@ def migrate_schema(db_password):
 
 def generate_passwords() -> Dict[str, str]:
     return {
-        "api": "",
+        "api": ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(50)),
         # "schema_migrator": ""
     }
 
 
 def set_passwords_for_db_users(passwords):
-    pass
+    conn_string = "host='localhost' port='5432' dbname='montagu' user='vimc' password='changeme'"
+    conn = psycopg2.connect(conn_string)
+    cur = conn.cursor()
+    cur.execute("ALTER USER vimc WITH PASSWORD '{}'".format(passwords["api"]))
+    conn.commit()
+    cur.close()
+    conn.close()
 
```

Comment:
Should there be some kind of try/finally logic around closing the cursor and connection? Or use Python's equivalent of C# using, which is `with psycopg2.connect(conn_string) as conn:`. I don't know if psycopg2 supports that, but if so, that would be the best I think.